<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeLineWidget</class>
 <widget class="QFrame" name="TimeLineWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>78</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="3">
    <widget class="QLabel" name="frameLabel">
     <property name="text">
      <string>Frame</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QSpinBox" name="startFrame">
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pauseButton">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="QDoubleSpinBox" name="fps">
     <property name="maximum">
      <double>60.000000000000000</double>
     </property>
     <property name="value">
      <double>25.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QLabel" name="endLabel">
     <property name="text">
      <string>End</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="gotoEndButton">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QSpinBox" name="endFrame">
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>250</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="playButton">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QLabel" name="fpsLabel">
     <property name="text">
      <string>FPS</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="startLabel">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QSpinBox" name="frame">
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="gotoStartButton">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="3">
    <widget class="QSlider" name="scrubber">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBothSides</enum>
     </property>
     <property name="tickInterval">
      <number>1</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
